### PR TITLE
Slack alert vanilla

### PR DESCRIPTION
**Pull request by ofiriro3**
### Summary of your changes
This tasks will create and enable alerts for the weekly environment.

**In-depth**
Currently, the job only deployed a KSPM vanilla environment on the Elastic Cloud and an agent on an EC2 machine.
Now, we wish to add alerts that will sent to a dedicated channel if any anomaly was detected.

The expected behavior is:
1. A new ELK environment is created with each execution.
2. A new Vanilla integration is installed on this environment.
3. The same EC2 machine whose credentials exist in the GitHub secret will deploy the agent.
4. A slack message will be sent to `ofir-personal-channel` with the relevant credentials for this environment.
**5. A new connector and a new rules were created in the Stack management section (In Kibana navigate to 'Stack Mangement' -> Alerts and insights -> Connectors/Rules).

**How to check the daily job**
Currently, the flow is deployed using a dispatch workflow, meaning - you will need to apply it using GitHub actions.
You can apply the flow from my fork, using [actions](https://github.com/ofiriro3/cloudbeat/actions/workflows/weekly-enviroment.yml).
Please set the branch to `slack-alert-vanilla` before running the dispatch workflow.
It should look like this:
image

The flow will deploy the integration to https://cloud.elastic.co/home, in the next PRs we will change it to a staging environment and replace the latest image with the SNAPSHOT version.

### Related Issues
Related https://github.com/elastic/security-team/issues/5790

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: .github/workflows/weekly-enviroment.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ on:
 env:
   WORKING_DIR: deploy/weekly-environment
   SCRIPTS_DIR: deploy/weekly-environment/scripts/benchmarks/kspm_vanilla
+  ALERTS_DIR: deploy/weekly-environment/scripts/alerts
+  HC_DIR: deploy/weekly-environment/scripts/health_checks
```
Comment (ofiriro3) [inline .github/workflows/weekly-enviroment.yml:27]: Not sure about the health_checks directory

File: deploy/weekly-environment/scripts/utils.sh [hunk 2]
```diff
@@ -133,6 +133,10 @@ create_new_vanilla_integration_manifest_file() {
   # get the item field from the response
   MANIFEST_FILE=$(echo "$manifest_creation_response" | jq -r '.item')
   echo "$MANIFEST_FILE" > manifest.yaml
+
+  # Update the imagePullPolicy to Always
+  yq e -i 'select(.kind == "DaemonSet").spec.template.spec.containers[0].imagePullPolicy = "Always"' manifest.yaml
```
Comment (olegsu) [inline deploy/weekly-environment/scripts/utils.sh:138]: Let's create kustomization directory with a patch.

File: .github/workflows/weekly-enviroment.yml (hunk 1 above)
Comment (olegsu) [reply .github/workflows/weekly-enviroment.yml:27]: When the health checks are called?

File: .github/workflows/weekly-enviroment.yml [hunk 3]
```diff
@@ -81,6 +84,21 @@ jobs:
...
           ssh -o StrictHostKeyChecking=no -v -i weekly-key.pem "ubuntu@${{ secrets.WEEKLY_ENVIRONMENT_EC2_PUBLIC_IP }}" "kubectl apply -f manifest.yaml"
 
+      - name: Check if the latest index was updated
+        working-directory: ${{ env.HC_DIR }}
+        env:
+          max_attempts: 10
+          retry_wait_seconds: 60
+        run: |
+          ./count_latest_index_result.sh ${{ env.ELASTICSEARCH_URL }} ${{ env.ELASTICSEARCH_PASSWORD }} ${{ env.max_attempts }} ${{ env.retry_wait_seconds }}
+
```
Comment (gurevichdmitry) [inline .github/workflows/weekly-enviroment.yml:94]: What will happen if "count_latest_index_result" script return failure? The workflow does not contain cleanup steps. Is it planned to add also environment cleanup?

**Review by gurevichdmitry — CHANGES_REQUESTED** [review]

**Comment by oren-zohar** [comment]
@kfirpeled @gurevichdmitry I'm closing this

**Pull request closed without merging**